vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -54,9 +54,12 @@ def from_config(cls, config: Dict[str, Any]) -> "CompressedTensorsConfig":
                 layer_quant_details[target][
```

Comment:
Before this function gets too crazy - can you add a comments about the structure of the `quant_config` that just covers what this logic is?

I know we are going to move this into `compressed-tensors` at some point...